packages/trader/src/App/Containers/Layout/install-pwa.jsx

```diff
@@ -1,6 +1,7 @@
 import { Button }       from 'deriv-components';
 import PropTypes        from 'prop-types';
 import React            from 'react';
+import { Button }       from 'deriv-components';
```

Comment:
why?

Reply:
It was added to get rid of eslint errors, but the import above it should be deleted :+1:

Reply:
there are two more files with button imports

Reply:
Yes, we overlooked this, will fix :pray: